docs_user/modules/ceph-mds_migration.adoc

```diff
@@ -33,7 +33,7 @@ We assume that:
 Before starting the MDS migration, verify the Ceph cluster is healthy and gather
 some information of the MDS status.
 
-[source,bash]
```

Comment:
I'm not sure if we want to remove the `source` here but i think we have to merge this because all patches are blocked on the docs breakage ( https://review.rdoproject.org/zuul/build/5262ed202d03415487a64b2777533e76 ). We can walk back this particular bit if needed. cc @klgill